src/AE2.lua

```diff
@@ -3,13 +3,20 @@ local ME = component.me_interface
 
 local AE2 = {}
 
-function AE2.requestItem(name, count)
+function AE2.requestItem(name, threshold, count)
     craftables = ME.getCraftables({
         ["label"] = name
     })
 
     if #craftables >= 1 then
         item = craftables[1].getItemStack()
+        itemInSystem = ME.getItemsInNetwork({
+        	["label"] = name        
+        })
+        if (#itemInSystem > 0 and itemInSystem[1]["size"] > threshold) then 
+            print("The amount of " .. itemInSystem[1]["label"] .. " exceeds threshold! Aborting request.")
```

Comment:
Put the print with time like in Run.lua in a function in maybe Utilities.lua (I can do that as well) so it can be used wherever to keep consitency on OC screen.

Reply:
I'm not really a big fan of having print statements in functions like these so in commit ab04f175931301f0809796eb65b39ce0ab47efcb i made it so the function returns an unpacked table of whether the craft request succeeded and a relevant message to go along with it. I believe this should make adding more messages easier.
Example:
 ![image](https://github.com/user-attachments/assets/a9b2a88a-34d3-427e-ba7c-4a376a626bd2)